python/cugraph/cugraph/tests/structure/test_graph_mg.py

```diff
@@ -234,6 +238,8 @@ def test_create_graph_with_edge_ids(dask_client, graph_file):
     )
 
 
+@pytest.mark.mg
+@pytest.mark.mg_test
```

Comment:
I think this needs to be removed since I don't see it in `pytest.ini`, look like it was leftover from a previous commit?

Reply:
we started with mg_test and then dropped to just mg.  Not sure why this one was not just edited.